### PR TITLE
Feature/2537/react to given reactions

### LINKED ISSUE #2537: Too hard to react with already given reactions
### Steps to reproduce

1. See a message with many different reactions
Screenshot_2022-11-04-11-52-10-006_com nextcloud talk2

2. Try to react with them

### Expected behaviour

It should be easily possible to react with already available reactions

### Actual behaviour

You can only react with a default list of reactions or you need to know the name of the emoji and use the search
Screenshot_2022-11-04-11-53-37-879_com nextcloud talk2

### Device brand and model

Xiaomi RedMi Note 8

### Android version

11

### Nextcloud Talk app version

Beta

### Nextcloud server version

25.0.1

### Talk version

15.0.1

### Custom Signaling server configured

_No response_

### Custom TURN server configured

_No response_

### Custom STUN server configured

_No response_

### Android logs

_No response_

### Server log

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by mahibi**
resolve #2537

known issue: #2603 (will be solved later)

### CODE REVIEW

**Review by AndyScherzinger — APPROVED** [review]
works like a charm @mahibi 💙

File: app/src/main/java/com/nextcloud/talk/ui/dialog/MessageActionsDialog.kt [hunk 1]
```diff
@@ -302,88 +306,70 @@ class MessageActionsDialog(
...
+        }
+
+        override fun onComplete() {
+            dismiss()
+        }
     }
 
     companion object {
-        private const val TAG = "MessageActionsDialog"
+        private val TAG = MessageActionsDialog::class.java.simpleName
```
Comment (AndyScherzinger) [inline app/src/main/java/com/nextcloud/talk/ui/dialog/MessageActionsDialog.kt:370]: why not a `const` anymore?

**Comment by AndyScherzinger** [comment]
@mahibi extra praise for removing the `Magic`prefix on some classes 🥇

File: app/src/main/java/com/nextcloud/talk/ui/dialog/MessageActionsDialog.kt (hunk 1 above)
Comment (mahibi) [reply app/src/main/java/com/nextcloud/talk/ui/dialog/MessageActionsDialog.kt:370]: it can't be a `const` when using `::class.java.simpleName`

**Pull request merged**